sdk/storage/azure-storage-blob/azure/storage/blob/_shared_access_signature.py

```diff
@@ -187,7 +193,7 @@ def generate_container(self, container_name, permission=None, expiry=None,
 
 
 class _BlobSharedAccessHelper(_SharedAccessHelper):
-    def __init__(self):
+    def __init__(self):  # pylint: disable=useless-super-delegation
```

Comment:
why disable this?

Reply:
pylint should rather throw a "useless constructor" here. :\

Reply:
Constructors deleted. Someone opened up an issue with pylint for that reason, but it looks like it was closed. :/